### PR TITLE
VB-3756, add day of week to prisoner profile-visits

### PR DESCRIPTION
**Pull request by hutcheonb-moj**
Add day of week to the date for each visit on prisoner profile page

### CODE REVIEW

File: integration_tests/integration/prisonerProfile.cy.ts [hunk 1]
```diff
@@ -96,11 +96,7 @@ context('Prisoner profile page', () => {
       .should('have.attr', 'href', `/visit/${profile.visits[0].reference}`)
     prisonerProfilePage.visitTabType().eq(0).contains('Social')
     prisonerProfilePage.visitTabLocation().eq(0).contains('Hewell (HMP)')
-    prisonerProfilePage
-      .visitTabDateAndTime()
-      .eq(0)
-      .contains(format(new Date(profile.visits[0].startTimestamp), prettyDateFormat))
-      .contains('10am - 11am')
```
Comment (dhirajshettymoj) [inline integration_tests/integration/prisonerProfile.cy.ts:103]: should we keep the contains for time?

Comment (hutcheonb-moj) [reply integration_tests/integration/prisonerProfile.cy.ts:103]: its still there - just one line

Comment (hutcheonb-moj) [reply integration_tests/integration/prisonerProfile.cy.ts:103]: well spotted - done

**Review by dhirajshettymoj — APPROVED** [review]
LGTM!

**Pull request merged**